docs/smart-agent-quickstart.md

```diff
@@ -1,132 +1,311 @@
+
+- internalStatusHost - This is the hostname that the Smart Agent will listen on so that the signalfx-agent status command can read diagnostic information from a running agent. This is also the host name the agent will listen on to serve internal metrics about the Smart Agent. These metrics can be scraped by the internal-metrics monitor. This will default to localhost if left blank.
+
+- internalStatusPort - This is the port that the Smart Agent will listen on so that the signalfx-agentstatus command can read diagnostic information from a running agent. This is also the host name the Smart Agent will listen on to serve internal metrics about the Smart Agent. These metrics can be scraped by the internal-metrics monitor. This will default to 8095.
+
+See the section on [Privileges](#Privileges) for information on the capabilities the Smart Agent requires.
+
+
+3. Run the Smart Agent by invoking the Smart Agent executable 
+
+```sh
+SignalFxAgent\bin\signalfx-agent.exe-config <path to config.yaml>. 
 
 ```
-sudo signalfx-agent status
+By default, the Smart Agent logs only to stdout/err. If you want to persist logs, you must direct the output to a log file or other log management system. See the [signalfx-agent command doc](https://github.com/signalfx/signalfx-agent/blob/master/docs/signalfx-agent.1.man) for more information on supported command flags.
+
+You may optionally install the Smart Agent as a Windows service by invoking the agent executable and specifying a few command line flags. The examples below show how to do install and start the Smart Agent as a Windows service.
+
+_Install Service_
+
+```sh
+PS> SignalFx\SignalFxAgent\bin\signalfx-agent.exe -service "install" -logEvents -config <path to config file>
+````
+
+_Start Service_
+
+```sh
+PS> SignalFx\SignalFxAgent\bin\signalfx-agent.exe -service "start"
+````
+
+
+
+### Privileges 
+
+_Linux_ 
+
+When using the host observer, the Smart Agent requires the Linux capabilities DAC\_READ\_SEARCH and SYS\_PTRACE, both of which are necessary to allow the agent to determine which processes are listening on network ports on the host. Otherwise, there is nothing built into the Smart Agent that requires privileges. 
+
+When using a package to install the Smart Agent, the Smart Agent binary is given those capabilities in the package post-install script, but the Smart Agent is run as the signalfx-agent user. If you are not using the host observer, then you can strip those capabilities from the Smart Agent binary if so desired.
+
+You should generally not run the Smart Agent as root unless you can’t use capabilities for some reason.
+
+_Windows_ 
+
+On Windows the Smart Agent must be installed and run under an administrator account.
+
+### Configuration 
```

Comment:
Seems like this should go with the configuration for Monitors and Observers. Everything you can configure in the yaml in one place?

Reply:
Moved to just before the Add Monitors topic. Done.